insertBaseData-psql.sql

```diff
@@ -1053,7 +1054,9 @@ INSERT INTO "authorizationGroups" (uuid, name, description, status, "createdAt",
   ('39a78d51-c351-452c-9206-4305ec8dd76d', 'EF 2.1', 'The complete EF 2.1 organisation', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
   ('c21e7321-7ec5-4837-8805-a302f9575754', 'EF 2.2', 'The complete EF 2.2 organisation', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
   ('ab1a7d99-4529-44b1-a118-bdee3ca8296b', 'EF 5', 'The complete EF 5 organization', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
-  ('90a5196d-acf3-4a81-8ff9-3a8c7acabdf3', 'Inactive positions', 'Inactive positions', 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+  ('90a5196d-acf4-4a82-8ff9-3a8c7acabdf4', 'Unlimited exporters', 'Unlimited exporters', 0, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
```

Comment:
In general, you should run a command like `uuidgen` and use that as a new uuid, instead of taking an existing uuid and manually editing that.